interface/super/edit_globals.php

```diff
@@ -645,6 +645,38 @@ function checkBackgroundServices()
                                                     echo "</option>\n";
                                                 }
                                                 echo "  </select>\n";
+                                            } elseif ($fldtype == GlobalSetting::DATA_TYPE_MULTI_DASHBOARD_CARDS) {
+                                                $hiddenList = [];
+                                                $ret = sqlStatement("SELECT gl_value FROM `globals` WHERE `gl_name` = 'hide_dashboard_cards'");
+                                                while ($row = sqlFetchArray($ret)) {
+                                                    $hiddenList[] = $row['gl_value'];
+                                                }
+                                                // The list of cards to hide. For now add to array new cards.
+                                                $res = array(
+                                                    ['card_abrev' => '', 'card_name' => xlt('None or Reset')],
```

Comment:
how does this work for a multiselect? Couldn't they select this and still select other items?